src/app/side-bar/list/list-header/list-header.component.html

```diff
@@ -0,0 +1,17 @@
+<div fxLayout="row" fxLayoutAlign="space-between" class="header-container">
+  <div fxlayout="row" fxLayoutAlign="start center" fxLayoutGap="16px">
+    <span><i class="pi pi-bars "></i>
+      <i class="pi pi-angle-left"></i>
+    </span>
+    <h4>Manage List</h4>
+  </div>
+  <div fxLayout="row" fxLayoutAlign="end center" fxLayoutGap="16px" class="header-container--btn">
+    <span><b>8752</b>Avaliable Credits</span>
```

Comment:
please give some spacing between **8752** and **Avaliable Credits**

Reply:
resolved ....................Thank you